src/main/logger/index.ts

```diff
@@ -1 +1,21 @@
-export * from "./lodestar";
+import electronLog from "electron-log";
+
+export const createLogger = (name: string, file: string): electronLog.ElectronLog => {
+    const logger = electronLog.create(name);
+    logger.transports.file.fileName = file;
+
+    if (process.env.NODE_ENV === "production") logger.transports.console.level = false;
+
+    return logger;
+};
+
+export const mainLogger = createLogger("mainLogger", "core.log");
+
+export const chainGuardianLogger = createLogger("chainGuardian", "chainGuardian.log");
```

Comment:
maybe some shorter names for usage like. mainLog, appLog or something?

Reply:
keeping a short as `log` can be confusing but can change `chainGuardianLogger` to `cgLogger`